test/testsuites/operatorhubtests.go

```diff
@@ -90,18 +90,12 @@ func testDisableAllEnableOne(t *testing.T) {
 	err = toggle(t, 1, false, true)
 	require.NoError(t, err, "Error updating cluster OperatorHub")
 
+	err = checkCreated(1, namespace)
```

Comment:
I guess this technically works but the point of this test case is to make sure only the sources we disabled are absent from the cluster, and the one we didn't disable, is present on the cluster.